src/app/helpers/jwt.interceptor.spec.ts

```diff
@@ -1,45 +1,48 @@
-import {TestBed} from '@angular/core/testing';
-import {HttpClientTestingModule} from '@angular/common/http/testing';
-import {httpHandlerSpy, httpRequestSpy} from './spies';
-import {HttpHandler, HttpRequest} from '@angular/common/http';
-import {of} from 'rxjs';
-import {JwtInterceptor} from './jwt.interceptor';
+import { TestBed } from "@angular/core/testing";
+import { HttpClientTestingModule } from "@angular/common/http/testing";
+import { httpHandlerSpy, httpRequestSpy } from "./spies";
+import { HttpHandler, HttpRequest } from "@angular/common/http";
+import { of } from "rxjs";
+import { JwtInterceptor } from "./jwt.interceptor";
+import { AuthService } from "angularx-social-login";
+import { authServiceSpy } from "./social.spies";
 
+const mockAuthService = jasmine.createSpyObj(['isLoggedIn']);
 
-describe('JWTInterceptor', () => {
+describe("JWTInterceptor", () => {
```

Comment:
Preference for single quotes over double quotes was agreed upon by the team for `.ts` files. You might want to set that default on your editor to avoid these changes.

Reply:
@wangonya I have fixed this as well.